lib/ansible/module_utils/keycloak.py

```diff
@@ -339,3 +380,125 @@ def delete_client_template(self, id, realm="master"):
         except Exception as e:
             self.module.fail_json(msg='Could not delete client template %s in realm %s: %s'
                                       % (id, realm, str(e)))
+
+    def get_scope_mappings(self, id, target='client', realm='master'):
+        """
+        Get scope mappings for client or client template
+
+        :param id: id of client or client template
+        :param target: either client or client-template
+        :param realm: realm of the client or client template
+        :return: dict of mappings with keys 'clientMappings' and 'realmMappings'
+        """
+        url = URL_CLIENT_SCOPE_MAPPINGS.format(url=self.baseurl, realm=realm, id=id, target=target)
+
+        try:
+            return json.load(open_url(url, method='GET', headers=self.restheaders,
+                                      validate_certs=self.validate_certs))
+        except ValueError as e:
+            self.module.fail_json(msg='API returned incorrect JSON when trying to obtain scope mappings for %s in realm %s: %s'
+                                      % (id, realm, str(e)))
```

Comment:
Would do this as a separate PR after keycloak_realm is in so as to avoid merge conflicts with the import (and then the entire rest of the file can be fixed too).

Reply:
Okay, thanks, sounds good. :-)